packages/kit/src/exports/public.d.ts

```diff
@@ -1012,6 +1012,10 @@ export interface RequestEvent<
 	 * related to the data request in this case. Use this property instead if the distinction is important to you.
 	 */
 	isDataRequest: boolean;
+	/**
+	 * `true` for `+server.js` calls coming from SvelteKit without the overhead of actually making an HTTP request. This happens when you make relative `fetch` requests on the server.
```

Comment:
I'd consider a fetch to an absolute path but no origin to be a relative fetch. It's not a full absolute URL - it's a URI relative to another URL.

Reply:
There are other issues for allowing overriding the origin of `event.url` in dev mode. I don't think we need to tie this to that at all.

In the `fetch` implementation in `packages/kit/src/runtime/server/fetch.js`, we're actually just checking whether `url.origin` equals `event.url.origin`. The URL that's being fetched has already been normalized at this point, and we can't tell whether it was called with a literal relative path or whether it was called with an origin that happened to match. So, maybe 'some-origin' would be a slightly more correct word to use here in the docs than 'relative'?

Reply:
'some-origin' looks so close to 'same-origin' I can't tell if it's a typo or not :laughing: I wonder if it'd be better to refer to host or hostname as more beginners might be familiar with it than origin and you can't specific an origin without specifying a hostname, so I think it's just as correct. Maybe something like:
```suggestion
	 * `true` for `+server.js` calls coming from SvelteKit without the overhead of actually making an HTTP request. This happens when you make `fetch` requests with the path, but not the host, on the server.
```

Reply:
lol yes that was a typo. I meant to say 'same-origin'.

What I was trying to get at was that if you explicitly `fetch` a full URL with an origin then we'll still do a subrequest if it matches the `event.url.origin`. That's what I was trying to get at with 'same-origin' - it doesn't matter whether it was explicitly same-origin or implicitly same-origin.

I think I wouldn't worry _too_ much about using a term like 'same-origin' since that's something that's going to come up all over the place in your web development career - it's not at all a niche thing.

Reply:
Oh, okay. I thought same-origin requests would be false unless it was a relative fetch. So yeah, in that case we should definitely update this! And in that case "same-origin" does indeed sound fine to me. I actually did think you might have meant "some" origin as in non-relative or something :laughing: 
```suggestion
	 * `true` for `+server.js` calls coming from SvelteKit without the overhead of actually making an HTTP request. This happens when you make same-origin `fetch` requests on the server.
```

Reply:
:+1: Updated.